resources/measures/ResidentialHVACCoolingSetpoints/measure.rb

```diff
@@ -116,8 +116,14 @@ def arguments(model)
     season_end_month.setDescription("End month of the cooling season.")
     season_end_month.setDefaultValue("Dec")
     args << season_end_month
-    
-    #TODO: Add thermostat deadband argument
+
+    # Make a string argument for thermostat deadband
+    onoff_thermostat_deadband = OpenStudio::Measure::OSArgument::makeStringArgument("onoff_thermostat_deadband", true)
```

Comment:
Is there a reason to use a string argument instead of a double argument?

Reply:
Resolved